kubewarden-dashboard.json

```diff
@@ -65,8 +23,8 @@
   "fiscalYearStartMonth": 0,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": null,
```

Comment:
What is `id` and why is 26 the right value ?

Reply:
I do not know. I believe Grafana did this change when I've exported the dashboard.